Code/GraphMol/MolStandardize/Wrap/testMolStandardize.py

```diff
@@ -228,9 +229,9 @@ def test13Tautomers(self):
     ctaut = enumerator.Canonicalize(m)
     self.assertEqual(Chem.MolToSmiles(ctaut), "O=C1CCCCC1")
 
-    tauts = enumerator.Enumerate(m)
-    self.assertEqual(len(tauts), 2)
-    ctauts = list(sorted(Chem.MolToSmiles(x) for x in tauts))
+    taut_res = enumerator.Enumerate(m)
```

Comment:
In python land, you might be able to make "taut_res" itself iterable, this would perhaps prevent breaking code in python land.

You could also do the same in C++ land by subclassing std::vector<ROMOL_SPTR> but that has its own issues.